### ISSUE TITLE
Duplicate field name in error message

### ISSUE THREAD

**Issue by mayur-tolexo** [asaskevich/govalidator#251]
I have this type of structure
```javascript
//CreateRole : Create Role
type CreateRole struct {
	Role       CreateRoleInfo `json:"role" valid:"required"`
}
```

//CreateRoleInfo : create role request format as in current system
```javascript
type CreateRoleInfo struct {
	FkUserID  int    `json:"fk_seller_id" valid:"required"`
	Name      string `json:"name" valid:"required"`
	Type      string `json:"type" valid:"optional"`
	Status    string `json:"status" valid:"optional"`
}
```

I am sending the following as request:
```javascript
"role":{
		"name":"selleruserrole4",
		"status":"active",
		"type":"selleruser"
	}
```
```
and getting an error:
# fk_seller_id: non zero value required;;fk_seller_id: non zero value required;;
```

Why am I getting fk_seller_id twice in the error message?

**Comment by jasonlam604** [comment]
I'm seeing this instead
`false fk_seller_id: non zero value required;role: non zero value required`

Same input:
`t:= &CreateRole{Role : CreateRoleInfo {Name: "selleruserrole4",Type: "active",Status: "selleruser"}}`

**Comment by asaskevich** [comment]
Thank you for your investigation, it seems to be fixed in #330
